### DISCUSSION TITLE
Portable version

### DISCUSSION THREAD

**Discussion by oranwutang** [posit-dev/positron#3733] [Feedback]
Hi!
Is it possible for you to make a portable version of Positron that could be installed without administrator permissions?

Thanks!

**Comment by petetronic** [comment]
Thanks for checking out Positron! Which platform(s) were you looking at? For Windows, perhaps related is https://github.com/posit-dev/positron/issues/3703

**Reply by iainmwallace** [reply]
It would be great to have non admin installation possible on Mac also

**Reply by RoyiAvital** [reply]
Being portable is more like https://github.com/posit-dev/positron/issues/3932.

**Comment by oranwutang** [comment]
I was thinking on a portable version for windows, so you're right, this post is basically the same as #3703 (I swear I made a search before posting this one 🤐).

Anyways a portable version for Mac would be desirable. 

I'm not sure if there is such  kind of thing like portable apps in Linux, maybe something like "snaps"?
